Azure/azure-cli
# [Security] Add allowed_connections commands

**Description**  
[Security] Add allowed_connections commands

Add allowed_connections commands, list of all possible traffic between resources for the subscription and location.

**Testing Guide**  
Users can get a list, single (show) of list of all possible traffic between resources:

az security allowed_connections list
az security allowed_connections show --name Internal --resource-group mygroup

Tests are included, use:  azdev test security test_allowed_connections --live
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [*] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [*] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).
